crypto-keys/src/main/java/com/palantir/crypto2/keys/serialization/KeyMaterials.java

```diff
@@ -43,6 +44,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(KeyMaterials.class);
     private static final Map<Integer, ? extends KeySerializer> SERIALIZERS = KeySerializers.getSerializers();
+    private static final Map<Integer, ? extends SymmetricKeySerializer> SYMMETRIC_SERIALIZERS =
```

Comment:
call the one above ASYMMETRIC_SERIALIZERS?

Reply:
done